patches/@wordpress+block-editor+14.17.0.patch

```diff
@@ -1,8 +1,13 @@
 diff --git a/node_modules/@wordpress/block-editor/build-module/components/inserter/index.js b/node_modules/@wordpress/block-editor/build-module/components/inserter/index.js
-index 78b9b8f..3c0049d 100644
+index 78b9b8f..9a8efb8 100644
 --- a/node_modules/@wordpress/block-editor/build-module/components/inserter/index.js
 +++ b/node_modules/@wordpress/block-editor/build-module/components/inserter/index.js
-@@ -191,6 +191,7 @@ class Inserter extends Component {
+@@ -187,10 +187,12 @@ class Inserter extends Component {
+         'is-quick': isQuick
+       }),
+       popoverProps: {
++        ...this.props.popoverProps,
```

Comment:
Patch to allow improving screen reader accessibility through marking the block inserter as a dialog.